configs/configuration.yaml

```
@@ -0,0 +1,26 @@
+name: configuration 
```

```suggestion
name: cytopipe_default 
```

This is a more descriptive name :)